config_vsc.py

```diff
@@ -34,13 +47,13 @@
 for version in ['2018a', '2019b', '2021a']:
     genius_modulepath.append(f'/apps/leuven/skylake/{version}/modules/all')
 
-# Specify hortense access flag in order to run jobs
+# Specify dodrio access flag in order to run jobs
```

Comment:
the official name is `hortense`, i think it's better to keep the official name?

Reply:
ok